src/internal/views/nunjucks/charge-information/view.njk

```diff
@@ -174,23 +184,23 @@ formErrorSummary %}
   {% if isEditable %}
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full">
-        <form method="post" action="/licences/{{licenceId}}/charge-information/check" novalidate>
+        <form method="post" action="/licences/{{licenceId}}/charge-information/check" novalidate="novalidate">
```

Comment:
I thought `novalidate` is a boolean, so `<form .... novalidate>` should do the trick?

Reply:
I think it is valid HTML5 so will leave it for now:
https://www.w3.org/TR/2008/WD-html5-20080610/semantics.html